packages/mui-system/src/colorManipulator.js

```diff
@@ -348,3 +348,30 @@ export function private_safeEmphasize(color, coefficient, warning) {
     return color;
   }
 }
+
+/**
+ * Blend a transparent overlay color with a background color, resulting in a single
+ * RGB color.
+ * @param {string} background - CSS color
+ * @param {string} overlay - CSS color
+ * @param {number} opacity - Opacity multiplier in the range 0 - 1
+ * @param {number} [gamma=1.0] - Gamma correction factor. For gamma-correct blending, 2.2 is usual.
+ */
+export function blend(background, overlay, opacity, gamma = 1.0) {
```

Comment:
Should we export this as 

```suggestion
export function unstable_blend(background, overlay, opacity, gamma = 1.0) {


```

?

With #40958, assuming we can use https://caniuse.com/?search=color-mix, I'm not clear why we need to make this part of the public API.